NAMESPACE

```diff
@@ -1,7 +1,22 @@
 # Generated by roxygen2: do not edit by hand
 
-export(get_athena_query_response)
+export("%>%")
+export(convert_athena_type_to_arrow)
```

Comment:
The NAMESPACE is for making functions user-facing. I think `convert_athena_type_to_arrow` should be a private function, and any other private functions can be removed.

Reply:
[2ae0bf5](https://github.com/moj-analytical-services/dbtools/pull/30/commits/2ae0bf565e446eaba537c2284c3eea09b846a725)